jobs/ci-kubernetes-kubemark-gce-scale.sh

```diff
@@ -64,8 +64,6 @@ export KUBE_NODE_OS_DISTRIBUTION="gci"
 
 # TODO: revert after running experiments.
 export EVENT_PD="true"
-# TODO remove after #19188 is fixed
-export CUSTOM_ADMISSION_PLUGINS="NamespaceLifecycle,LimitRanger,ResourceQuota"
```

Comment:
@wojtek-t Just curious why we need to even remove the NamespaceLifecycle plugin? :)
Would it also affect the performance of secrets in some way?